gravitee-apim-console-webui/src/management/settings/api-portal-header/migrated/api-portal-header-routing.modile.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { ApiPortalHeaderComponent } from "./api-portal-header.component";
+
+const routes: Routes = [
+  {
+    path: '',
+    component: ApiPortalHeaderComponent
+  }
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule]
+})
+export class ApiPortalHeaderRoutingModule { }
```

Comment:
We don't usually create routing files for each page, but keep 3 or 4 main ones in the application 
For this page it will be `settings.route.ts` which combines the routing of environment settings